stable/odoo/templates/deployment.yaml

```diff
@@ -26,14 +26,33 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: POSTGRESQL_HOST
+        {{- if .Values.postgresql.enabled }}
```

Comment:
Instead of these if/else you could use the `ternary` operator

Reply:
Done